index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
```

Comment:
having 3 consts called calc, c, and calculator is perhaps a bit confusing, my suggested renaming would be:

- calc to calculator, since this is what it is - the calculator file
- c to something like userInput
- calculator to (...tough one) not sure...